docs/how-to/tuning-guides/mi300x.rst

```diff
@@ -0,0 +1,1429 @@
+      be ongoing before moving on to the next step. For example, if ``n`` is
+      ``0``, wait for all ``lgkm`` access to finish before continuing. If ``n``
+      is ``1``, move on even if ``1`` ``lgkm`` access is still running
+      asynchronously.
+
+   *  ``vmcnt(n)``: ``vm`` represents vector memory. This happens when
+      vector memory is accessed, for example, when global load moves
+      from global memory to vector memory. The variable ``n`` is the same as
+      the previous setting.
+
+Generally recommended guidelines are as follows.
+
+*  Vectorize memory access as much as possible.
+
+*  Ensure synchronization is done efficiently.
+
+*  Overlap of instructions to hide latency, but it requires thoughtful
+   analysis of the algorithms.
+
+*  If you find inefficiencies, you can trace it back to LLVM IR, TTGIR
+   and even TTIR to see where the problem comes from. If you find it
+   during compiler optimization, activate the MLIR dump
+   (``export MLIR_ENABLE_DUMP=1``) and check which optimization pass caused the
+   problem.
+
+PyTorch inductor Triton tuning knobs
+====================================
+
+The following are suggestions for optimizing matrix multiplication (GEMM) and
+convolution (``conv``) operations in PyTorch using ``inductor``, a part of the
+PyTorch compilation framework. The goal is to leverage Triton to achieve better
+performance.
+
+Learn more about TorchInductor environment variables and usage in
+`PyTorch documentation <https://pytorch.org/docs/2.3/torch.compiler_inductor_profiling.html>`_.
+
+To tune Triton kernels with ``gemm`` and convolution ops (``conv``), use the
+``torch.compile`` function with the ``max-autotune`` mode. This benchmarks a
+predefined list of Triton configurations and selects the fastest one for each
+shape. See the configurations in PyTorch source code:
+
+* `conv configs for max-autotune <https://github.com/pytorch/pytorch/blob/a1d02b423c6b4ccacd25ebe86de43f650463bbc6/torch/_inductor/kernel/conv.py#L51>`_
+
+* `matmul configs for max-autotune <https://github.com/pytorch/pytorch/blob/a1d02b423c6b4ccacd25ebe86de43f650463bbc6/torch/_inductor/kernel/mm_common.py#L118>`_
+
+.. note::
+   Triton is not used if regular :doc:`MIOpen <miopen:index>` or
+   :doc:`rocBLAS <rocblas:index>` performs faster for a specific operation.
+
+* Set ``torch._inductor.config.max_autotune = True`` or ``TORCHINDUCTOR_MAX_AUTOTUNE=1``.
+
+* Or, for more fine-grained control:
+
+  ``torch._inductor.config.max_autotune_gemm = True``
+     To enable tuning or lowering of ``mm``/``conv``\s.
+
+  ``torch._inductor.config.max_autotune.pointwise = True``
+     To enable tuning for ``pointwise``/``reduction`` ops.
+
+  ``torch._inductor.max_autotune_gemm_backends`` or ``TORCHINDUCTOR_MAX_AUTOTUNE_GEMM_BACKENDS``
+     Selects the candidate backends for ``mm`` auto-tuning. Defaults to
+     ``TRITON,ATEN``. This also includes the ``CUTLASS`` tuning option for
```

Comment:
Can you remove the sentence of "This also include the CUTLASS tuning option for NVIDIA CUDA". We don't need to mention NVIDIA in this document.